pkgs/applications/audio/midi-trigger/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, stdenv, fetchFromGitHub, pkg-config, lv2 }:
+stdenv.mkDerivation rec {
```

Comment:
```suggestion
{ lib, gccStdenv, fetchFromGitHub, pkg-config, lv2 }:

gccStdenv.mkDerivation rec {
```

Reply:
@SuperSandro2000 does this mean it will always be built using GCC? Currently it depends on the platform and on Darwin it’s built using Clang.

Reply:
I mean when you change `stdenv` to `gccStdenv` I just assume by the name it’s always using GCC.

Reply:
Yeah, then gcc would be used in darwin, too.

Reply:
@SuperSandro2000 then maybe let’s keep `stdenv`? The app supports builds both with `clang` and `gcc`. Let’s keep native compiler for the platform?